web3.js/test/compute-budget.test.ts

```diff
@@ -0,0 +1,160 @@
+import {expect, use} from 'chai';
+import chaiAsPromised from 'chai-as-promised';
+
+import {
+  Keypair,
+  Connection,
+  LAMPORTS_PER_SOL,
+  Transaction,
+  ComputeBudget,
+  ComputeBudgetInstruction,
+  PublicKey,
+  SystemProgram,
+  sendAndConfirmTransaction,
+} from '../src';
+import {helpers} from './mocks/rpc-http';
+import {url} from './url';
+
+use(chaiAsPromised);
+
+describe.skip('ComputeBudget', () => {
```

Comment:
currently skipping these tests since I don't think that the transaction wide compute cap is activated on most networks yet?

Reply:
It's enabled by default in new runs of the test validator